client/my-sites/plan-testimonials/index.jsx

```diff
@@ -18,7 +18,7 @@ export class PlanTestimonials extends Component {
 				</div>
 				<div className="plan-testimonials__content-right-container">
 					<div className="plan-testimonials__content-right">
-						Did you know that WordPress powers 36% of the entire internet? WordPress.com is the best
+						Did you know that WordPress powers 35% of the entire internet? WordPress.com is the best
```

Comment:
It's a bit off to me that this component is wrapped by `localize()` yet this whole copy is not wrapped by `translate()`. Do you know if that's intentional?

@akirk I'd love to learn your opinion on this as well, in case there is any context that we are missing :)

Reply:
It's a redundancy that could be avoided. The `localize()` is not necessary here, Since this testimonials section is a new component being A/B tested, we can make this change soon enough when the test ends. @akirk had mentioned this too in the [slack chat](https://a8c.slack.com/archives/C22KJ8T6C/p1583312882018700?thread_ts=1583229318.005800&cid=C22KJ8T6C)

Reply:
Cool. Thanks for clarifying.